ha/setup/pre_disruptive_upgrade.py

```diff
@@ -180,7 +181,8 @@ def _parse_arguments() -> Any:
 
 def _main() -> None:
     args = _parse_arguments()
-    Log.init(service_name="pre_disruptive_upgrade", log_path=RA_LOG_DIR, level="INFO")
+    ConfigManager.init(log_name="pre_disruptive_upgrade",
+                                        log_path=RA_LOG_DIR, level="INFO")
```

Comment:
SAme comment as before RA_LOG_DIR is still pointing to /var/log/seagate/cortx/ha

Reply:
similar as above

Reply:
same as previous comment